src/services/events/event-schema.ts

```diff
@@ -0,0 +1,49 @@
+import mongoose from "mongoose";
+import { z } from "zod";
+
+// Zod schema for event
+const EventValidator = z.object({
+    name: z.string(),
+    description: z.string(),
+    start_time: z.date(),
+    end_time: z.date(),
+    attendees: z.array(z.string()),
+    location: z.array(
+        z.object({
+            description: z.string(),
+            tags: z.array(z.string()),
+            latitude: z.number(),
+            longitude: z.number(),
+        })
+    ),
+    virtual: z.boolean(),
+    upgrade: z.boolean().default(false),
+    downgrade: z.boolean().default(false),
```

Comment:
let's remove these two lines and replace it with a point value for each event
```suggestion
    points: z.number().min(0).default(0),
```